pkg/db/pebble.go

```diff
@@ -94,14 +96,32 @@ func buildPebbleOption(id int, memInByte int, cfg *config.DBConfig) (pebble.Opti
 		atomic.StoreInt64(&stallTimeStamp, 0)
 		ws.duration.Store(time.Since(time.Unix(0, start)))
 	}
+
 	return option, ws
 }
 
 // OpenPebble opens a pebble.
 func OpenPebble(
 	ctx context.Context, id int, path string, memInByte int, cfg *config.DBConfig,
+) (DB, error) {
+	return doOpenPebble(ctx, id, path, memInByte, true, cfg)
+}
+
+func doOpenPebble(
+	ctx context.Context,
+	id int, path string, memInByte int,
+	withTablePropertyCollectors bool,
```

Comment:
seems `withTablePropertyCollectors` is always true, why we need this flag?

Reply:
For some tests in which keys are not valid cdc event key format.